drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/popup/FileUploadPopup.java

```diff
@@ -38,12 +38,16 @@ void show(final List<String> acceptedExtension,
                   final Command okCommand);
 
         String getFileContents();
+
+        String getFileName();
```

Comment:
I think it is better not to add the method if we do not need it currently. Can happen in final won't be needed at all but we will forget to remove it from API.

Reply:
@jomarko I was wrong, it is used is `ScenarioSimulationEditorKogitoTestingScreen` (present in branch 3879 only). Please ignore my previous comment.